### DISCUSSION TITLE
Can the server support custom code prompts?

### DISCUSSION THREAD

**Discussion by wrf1998** [TypeFox/monaco-languageclient#657] [Q&A]
I am currently using the python server in the example, and I find that its code hints only support the data provided by pyright, but I have some packages that are internal to the company, and I want to be able to hint and hover the mouse to see some specific information about the package, etc. How do I need to modify my server side

**Comment by kaisalmen** [comment]
See https://github.com/TypeFox/monaco-languageclient/issues/656#issuecomment-2106787659
